### PR TITLE
Added Android.mk to compile via NDK-build

### PR DESCRIPTION
**Pull request by Thanh9356**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-leap-day
```
Comment (Aurel300) [inline _config.yml]: Please remove this file from the PR.

File: Android.mk [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+src/std/string.c \
+src/std/sys.c \
+src/std/types.c \
+src/std/ucs2.c \
+src/std/thread.c \
+src/std/process.c \
+src/std/sys_android.c \
+src/alloc.c \
+src/hl.h \
+src/hlc.h \
```
Comment (Thanh9356) [inline Android.mk]: That's OK here

File: Android.mk [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                    $(LOCAL_PATH)/./include/mbedtls \
+                    $(LOCAL_PATH)/./include/mikktspace \
+                    $(LOCAL_PATH)/./include/openal\include \
+                    $(LOCAL_PATH)/./include/pcre \
+                    $(LOCAL_PATH)/./include/png \
+                    $(LOCAL_PATH)/./include/sdl/include \
+                    $(LOCAL_PATH)/./include/sqlite/src \
+					$(LOCAL_PATH)/./include/turbojpeg \
+					$(LOCAL_PATH)/./include/vorbis \
+					$(LOCAL_PATH)/./include/zlib \
```
Comment (Thanh9356) [inline Android.mk]: Anybody can help me this? Thanks a lot

Comment (josuigoa) [reply Android.mk]: Did you get working with this setup?

Comment (Thanh9356) [reply Android.mk]: Yes, I did...

**Pull request closed without merging**